.github/workflows/ci.yml

```diff
@@ -382,6 +386,15 @@ jobs:
         if: ${{ failure() || cancelled() || steps.analyze-test-run.outputs.flakyTests != '' }}
         with:
           name: "[UT] Zeebe - ${{ matrix.component }} - ${{matrix.suiteName}}"
+      - name: Write matrix outputs for test analysis
+        uses: cloudposse/github-action-matrix-outputs-write@v1
```

Comment:
Were you able to use this action in practice? According to our [usage of 3rd party GHA](https://github.com/camunda/camunda/wiki/CI-&-Automation#usage-of-third-party-github-actions) this should be prohibited from execution and is not on the documented allowlist in wiki.

Reply:
Yes, but only after adding it to the allow list. [Here](https://camunda.slack.com/archives/CHY2S7KDJ/p1754042035986739?thread_ts=1753914211.641289&cid=CHY2S7KDJ) the thread about it's usage. Nice catch about the wiki! Will update it.

Reply:
Ah I missed out on the fact that we allowed additional 3rd party GHAs in the repo already